test-specifications/crypto-service/accountCreateTransaction.md

```diff
@@ -256,12 +257,12 @@ https://docs.hedera.com/hedera/sdks-and-apis/rest-api
 
 - The maximum number of tokens with which an account can be implicitly associated. Defaults to 0 and up to a maximum value of 1000.
 
-| Test no | Name                                                                           | Input                                          | Expected response                                                                                                                | Implemented (Y/N) |
-|---------|--------------------------------------------------------------------------------|------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|-------------------|
-| 1       | Creates an account with a max token association that is a valid amount         | key=<VALID_KEY>, maxAutoTokenAssociations=100  | The account creation succeeds and the account has 100 automatic token associations.                                              | N                 |
-| 2       | Creates an account with a max token association that is the minimum value      | key=<VALID_KEY>, maxAutoTokenAssociations=0    | The account creation succeeds and the account has 0 automatic token associations.                                                | N                 |
-| 3       | Creates an account with a max token association that is the maximum value      | key=<VALID_KEY>, maxAutoTokenAssociations=1000 | The account creation succeeds and the account has 1000 automatic token associations.                                             | N                 |
-| 4       | Creates an account with a max token association that exceeds the maximum value | key=<VALID_KEY>, maxAutoTokenAssociations=1001 | The account creation fails with a REQUESTED_NUM_AUTOMATIC_ASSOCIATIONS_EXCEEDS_ASSOCIATION_LIMIT response code from the network. | N                 |
+| Test no | Name                                                                           | Input                                                                                                  | Expected response                                                                               | Implemented (Y/N) |
```

Comment:
Not sure when we should add case with `-1` value, but putting this comment as a reminder about such case.

Reply:
The `-1` case should generally be included for every signed numeric value.  The reason being that it's the longest encoded value in protobuf, and tests the binary maximum edge case.

Reply:
I mean in the context of HIP-904 (where the value `-1` is  permitted for `max_auto_associations`, and this value means “unlimited”).

Reply:
Created [this](https://github.com/hashgraph/hedera-sdk-tck/issues/229) to add them. Tests will need to be added here, `AccountUpdateTransaction`, and a couple contract transactions

Reply:
Thanks
cc @0xivanov